component/src/main/java/com/celements/photo/unpack/UnpackComponent.java

```diff
@@ -87,6 +90,12 @@ public String unzipFileToAttachment(XWikiAttachment zipSrcFile, String attName,
         LOGGER.error("Exception while unpacking zip", ioe);
       } catch (XWikiException xwe) {
         LOGGER.error("Exception while unpacking zip", xwe);
```

Comment:
Please use Java 7 multiple Exception handling if catch body is identical. e.g. } catch (IOException|XWikiException exp) {...